autorest/azure/environments.go

```diff
@@ -50,54 +50,56 @@ type ResourceIdentifier struct {
-	ServiceBusEndpoint           string             `json:"serviceBusEndpoint"`
-	BatchManagementEndpoint      string             `json:"batchManagementEndpoint"`
-	StorageEndpointSuffix        string             `json:"storageEndpointSuffix"`
-	SQLDatabaseDNSSuffix         string             `json:"sqlDatabaseDNSSuffix"`
-	TrafficManagerDNSSuffix      string             `json:"trafficManagerDNSSuffix"`
-	KeyVaultDNSSuffix            string             `json:"keyVaultDNSSuffix"`
-	ServiceBusEndpointSuffix     string             `json:"serviceBusEndpointSuffix"`
-	ServiceManagementVMDNSSuffix string             `json:"serviceManagementVMDNSSuffix"`
-	ResourceManagerVMDNSSuffix   string             `json:"resourceManagerVMDNSSuffix"`
-	ContainerRegistryDNSSuffix   string             `json:"containerRegistryDNSSuffix"`
-	CosmosDBDNSSuffix            string             `json:"cosmosDBDNSSuffix"`
-	TokenAudience                string             `json:"tokenAudience"`
-	ResourceIdentifiers          ResourceIdentifier `json:"resourceIdentifiers"`
+	Name                              string             `json:"name"`
+	ManagementPortalURL               string             `json:"managementPortalURL"`
+	PublishSettingsURL                string             `json:"publishSettingsURL"`
+	ServiceManagementEndpoint         string             `json:"serviceManagementEndpoint"`
+	ResourceManagerEndpoint           string             `json:"resourceManagerEndpoint"`
+	ActiveDirectoryEndpoint           string             `json:"activeDirectoryEndpoint"`
+	GalleryEndpoint                   string             `json:"galleryEndpoint"`
+	KeyVaultEndpoint                  string             `json:"keyVaultEndpoint"`
+	GraphEndpoint                     string             `json:"graphEndpoint"`
+	ServiceBusEndpoint                string             `json:"serviceBusEndpoint"`
+	BatchManagementEndpoint           string             `json:"batchManagementEndpoint"`
+	StorageEndpointSuffix             string             `json:"storageEndpointSuffix"`
+	SQLDatabaseDNSSuffix              string             `json:"sqlDatabaseDNSSuffix"`
+	TrafficManagerDNSSuffix           string             `json:"trafficManagerDNSSuffix"`
+	KeyVaultDNSSuffix                 string             `json:"keyVaultDNSSuffix"`
+	ServiceBusEndpointSuffix          string             `json:"serviceBusEndpointSuffix"`
+	ServiceManagementVMDNSSuffix      string             `json:"serviceManagementVMDNSSuffix"`
+	ResourceManagerVMDNSSuffix        string             `json:"resourceManagerVMDNSSuffix"`
+	ContainerRegistryDNSSuffix        string             `json:"containerRegistryDNSSuffix"`
+	CosmosDBDNSSuffix                 string             `json:"cosmosDBDNSSuffix"`
+	TokenAudience                     string             `json:"tokenAudience"`
+	APIManagementHostManagementSuffix string             `json:"apiManagementHostManagementSuffix"`
```

Comment:
This suffix is not only for `HostManagement` but also for `Portal`, etc.... You should use `APIManagementHostNameSuffix` instead.